NGitLab/RequestOptions.cs

```diff
@@ -32,6 +32,7 @@ public class RequestOptions
     public TimeSpan HttpClientTimeout { get; set; } = TimeSpan.FromMinutes(5);
 
     public string UserAgent { get; set; }
+    public WebProxy Proxy { get; set; } =null;
```

Comment:
You'll need to modify `NGitLab\PublicAPI.Unshipped.txt`, to document changes made to the public API (for the getter and setter). We recommend that you do so using VS2022's "Quick Actions and Refactorings..." rather than modifying the file by hand.